### PR TITLE
Added documentation for installing nightly builds.

### PR DESCRIPTION
**Pull request by activist**
Please review added section for installing nightly builds.

This can be added once the pull request https://github.com/highcharts/highcharts/pull/12587 is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ See [npm documentation](https://docs.npmjs.com/) on how to get started with npm.
 npm install --save highcharts
 ```
 
+#### Install nightly builds
+See [highcharts documentation](https://www.highcharts.com/docs/getting-started/install-from-npm) for installing nightly builds.
+
+Note that we do not recommend the use of the nightly build in production environments as these **may contain bugs and are not considered stable.**
```
Comment (bre1470) [inline README.md]: As we only have one `nightly`, it should read: `Note that we do not recommend the use of the nightly build in production environments as it **may contain bugs and is not considered stable.**`

Comment (activist) [reply README.md]: Thanks, forgot to commit in this repo. Should be fixed now.

Comment (bre1470) [reply README.md]: The bold text is still written in plural.

Comment (activist) [reply README.md]: @bre1470 It is changed to `it` as far as I can see. https://github.com/highcharts/highcharts-dist/pull/17/files

Did you look at an outdated version?

Comment (bre1470) [reply README.md]: Yes, sorry! Probably some caching glitch.

**Pull request merged**